app/components/Article/KeepGoing/keepGoing.module.css

```diff
@@ -4,3 +4,14 @@
   padding: var(--spacing-32);
   margin-bottom: var(--spacing-56);
 }
+
+@media (max-width: 640px) {
```

Comment:
I noticed that this `640px` value is being used in multiple media queries? (For example, also in this PR https://github.com/StampyAI/stampy-ui/pull/451). Would it make sense to define a variable that there is only a single spot to change and make it easier to stay consistent between the different media queries?
(I got this syntax from Claude, so apologies if it is a bit off 😆 )
```suggestion
@media (max-width: var(--mobile-breakpoint)) {
```

Reply:
I tried that, but it didn't want to work. There might be a clever way of doing it

Reply:
Ah okay, fair enough. Unfortunately, I don't know what this clever way is then 😅

Reply:
:(